components/Home/Inovadoras-nft-collection/Cover/index.js

```diff
@@ -3,12 +3,12 @@ import React from 'react'
 
 export default function Cover() {
   return (
-    <div className="md:ml-28">
+    <div className="">
```

Comment:
Cant we just remove the className atribute in this div since it is not being utilized anymore? 
Just code style

Reply:
Total sense !